assets/queries/terraform/alicloud/nas_file_system_not_encrypted/query.rego

```diff
@@ -0,0 +1,29 @@
+package Cx
```

Comment:
Not sure if this query is necessary since PR #5248 will cover it. We should discuss it.

Nevertheless, do not forget to define `searchLine`, please.